spec/rails_helper.rb

```diff
@@ -3,8 +3,12 @@
 require 'spec_helper'
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
+require 'coveralls'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+#Coveralls activation command
+Coveralls.wear!
```

Comment:
Do something like https://github.com/osulp/Scholars-Archive/blob/master/spec/support/coveralls.rb instead. Support files make for cleaner edits rather than constantly changing rails_helper.